cardano-cli/src/Cardano/CLI/EraBased/Run/Key.hs

```diff
@@ -56,8 +60,20 @@ import qualified Data.Text as Text
 import qualified Data.Text.Encoding as Text
 import           System.Exit (exitFailure)
 
-drepKeyEnvelopeDescr :: TextEnvelopeDescr
-drepKeyEnvelopeDescr = "Delegate Representative Verification Key"
+drepVkeyDesc :: TextEnvelopeDescr
```

Comment:
This is fine for now here but if these definitions grow we probably want a separate module for them.

Reply:
Yeah I was considering that too, but I'm usually not so fond of `constant` modules. Would you like me to introduce a new `cardano-cli/src/Cardano/CLI/EraBased/Run/Constants.hs` module?

Reply:
Maybe we could use `HasTextEnvelope` to provide a description for a type? Or do we foresee cases where we could have different descriptions for the same type?

Reply:
Good point @carbolymer, I created a follow-up issue to track this: https://github.com/input-output-hk/cardano-cli/issues/429 because there are `44` instances of `HasTextEnvelop` in cardano-api :exploding_head: so that ought to be a separate effort from this issue.